extension/dialog.css

```diff
@@ -12,6 +12,7 @@
 }
 
 html, body {
+    color-scheme: light dark;
```

Comment:
Could you put this under `:root` for consistency? Or even meta color-scheme.

Reply:
I hadn't been aware of the [`<meta name="color-scheme">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name), previously.  Do you prefer it declared with a `<meta>` element?  I'll replace it accordingly.